src/render/TerraformTemplate.js

```diff
@@ -8,7 +8,7 @@ const root = `{% for _block in components %}
 {% endfor %}`;
```

Comment:
rename to `Update TerraformTemplate with dynamic attribute block`

Reply:
done